glide.lock

```diff
@@ -166,10 +174,10 @@ imports:
   version: 7af7a1e09ba336d2ea14b1ce73bf693c6837dbf6
 - name: github.com/seccomp/libseccomp-golang
   version: f6ec81daf48e41bf48b475afc7fe06a26bfb72d1
-- name: github.com/Sirupsen/logrus
-  version: f006c2ac4710855cf0f916dd6b77acf6b048dc6e
 - name: github.com/sirupsen/logrus
   version: 89742aefa4b206dcf400792f3bd35b542998eb3b
+- name: github.com/Sirupsen/logrus
```

Comment:
Do we need to have these duplicated?

Reply:
Yep, sadly https://github.com/sirupsen/logrus/issues/570#issuecomment-313933276 and some projects are using the other one. My next PR will be updating all the dependencies.